pages/v3-migration.page.tsx

```diff
@@ -0,0 +1,170 @@
+import { MigrationListItemLoader } from 'src/modules/migration/MigrationListItemLoader';
+import { MigrationLists } from 'src/modules/migration/MigrationLists';
+import { MigrationTopPanel } from 'src/modules/migration/MigrationTopPanel';
+import { usePoolDataV3Subscription, useRootStore } from 'src/store/root';
+import {
+  selectV2UserSummaryAfterMigration,
+  selectV3UserSummary,
+  selectV3UserSummaryAfterMigration,
+} from 'src/store/v3MigrationSelectors';
+
+export default function V3Migration() {
+  const { loading } = useAppDataContext();
+  const { currentAccount, loading: web3Loading } = useWeb3Context();
+  const { isPermissionsLoading } = usePermissions();
+  const setCurrentMarketForMigration = useRootStore((state) => state.setCurrentMarketForMigration);
+
+  const currentTimeStamp = useCurrentTimestamp(5);
+
+  const v2UserSummaryAfterMigration = useRootStore((state) =>
+    selectV2UserSummaryAfterMigration(state, currentTimeStamp)
+  );
+
+  const v3UserSummaryAfterMigration = useRootStore((state) =>
+    selectV3UserSummaryAfterMigration(state, currentTimeStamp)
+  );
+
+  const v3UserSummaryBeforeMigration = useRootStore((state) =>
+    selectV3UserSummary(state, currentTimeStamp)
+  );
+
+  const { user, borrowPositions } = useUserReserves();
+
+  const {
+    toggleMigrationSelectedSupplyAsset: toggleSelectedSupplyPosition,
+    selectedMigrationSupplyAssets: selectedSupplyAssets,
+    toggleMigrationSelectedBorrowAsset: toggleSelectedBorrowPosition,
+    selectedMigrationBorrowAssets: selectedBorrowAssets,
+  } = useRootStore();
+
+  useEffect(() => {
+    if (setCurrentMarketForMigration) {
+      setCurrentMarketForMigration();
+    }
+  }, [setCurrentMarketForMigration]);
+
+  usePoolDataV3Subscription();
+
+  return (
+    <>
+      <MigrationTopPanel />
+      {currentAccount && !isPermissionsLoading ? (
+        <ContentContainer>
+          <MigrationLists
+            loading={loading}
+            totalSuppliesUSD={user.totalCollateralUSD}
+            totalBorrowsUSD={user.totalBorrowsUSD}
+            isSupplyPositionsAvailable={!!user.userReservesData.length}
+            isBorrowPositionsAvailable={!!borrowPositions.length}
+            onSelectAllSupplies={() =>
+              user.userReservesData.map((reserve) =>
+                toggleSelectedSupplyPosition(reserve.underlyingAsset)
+              )
+            }
+            onSelectAllBorrows={() =>
+              borrowPositions.map((reserve) =>
+                toggleSelectedBorrowPosition(reserve.underlyingAsset)
+              )
+            }
+            suppliesPositions={
+              <>
+                {loading ? (
+                  <>
+                    <MigrationListItemLoader />
+                    <MigrationListItemLoader />
+                  </>
+                ) : !!user.userReservesData.length ? (
+                  user.userReservesData.map((reserve) => (
```

Comment:
Instead of displaying all V2 supply and borrow positions, there should be a selector to only display assets where `underlyingAsset` is available in both the V2 and V3 markets